### PR TITLE
Add prettier as a project linter and format src/

### PR DESCRIPTION
**Pull request by shrugs**
I never enjoy throwing up a multi-thousand line change day one of a project, but a linter is a very good thing to have. I'm unopinionated about javascript format, but opinionated about there being _a_ format, you know?

I've configured `prettier` with create-react-app (hence the `.eslintrc` plugin) to format the code pre-commit. 

- `npm run lint:fix` formats the src files in-place
- `npm run lint:check` runs a CI-friendly check script
- we use `husky` to install a precommit hook in the local repository and run the precommit npm script on commits, which ends up calling the `lint:check` script

I did my best to look at the existing code and infer some style guidelines (see `.prettierrc.json`) and have categorically ignored the `components/` and `nockjs/` folders because components will be changing with the move to indigo and nockjs is a highly-unique format that I expect the author is happy with.

---

To review this PR, check out the first commit to see the basic changes. Then, for additional confidence, check out that specific commit and run `npm run lint:fix`. By diffing your staged changes with this PR, you can be certain that the only files changed in the second commit are lint-related fixes and the precommit installation

### CODE REVIEW

File: src/bridge/lib/keys.js [hunk 1]
```diff
@@ -22,26 +18,19 @@ const b64 = buf => {
...
-    : 62 === j
-    ? 45
-    : 126
-
-  return '0w' +
-    c.reduce((a, b, i) =>
-      String.fromCharCode(trans(b)) +
-      (( i && 0 === i % 5 ) ? '.' : '') + a,
-      '')
+  const trans = j => (10 > j ? j + 48 : 36 > j ? j + 87 : 62 > j ? j + 29 : 62 === j ? 45 : 126)
```
Comment (Fang-) [inline src/bridge/lib/keys.js]: This seems kind of unfortunate. ):

**Comment by shrugs** [comment]
@Fang- I've rewritten the two commits and rebased against mino - we also now force semicolon usage and restrict printing to <=80 characters 👍

File: src/bridge/lib/keys.js (hunk 1 above)
Comment (shrugs) [reply src/bridge/lib/keys.js]: yikes, yeah that's pretty unreadable. we can ignore lines with `// prettier-ignore`, so in the last commit here I've reverted that specific line to what it was previously

**Comment by shrugs** [comment]
... and rebase/pushed again because of https://github.com/urbit/bridge/pull/150 merge

**Comment by Fang-** [comment]
Sorry, I'm going to ask you to force-push this one more time. I want this to go in _on top of_ #151 instead of underneath it. Seems easier than wrestling trivial merge conflicts for half an hour. (Or is there a saner way to do this?) Either target that branch (`mino-send-invites`), or wait out its merge, whatever you feel is easiest.

(Wow I already hate this `prettier-ignore` fighting-the-tooling shit.)

**Comment by shrugs** [comment]
@Fang- oh yeah totally, let's wait for 151. just re-applying the `npm run lint:fix` and having everyone base off branch is definitely easiest.

I see you on the prettier-ignore. In prettier's defense that's a pretty gnarly nested ternary that fits nicely within 80 characters 😂

**Comment by shrugs** [comment]
@Fang- updated after the recent merge! I'd also recommend configuring your editor to run prettier on save so we never actually have to run the npm scripts

**Pull request merged**